packages/api/src/typeDefs/collection.graphql

```diff
@@ -11,13 +11,21 @@ enum StoreCollectionType {
   """
```

Comment:
Could you include these new ones in this list as well?
https://github.com/vtex/faststore/blob/5102824853b33544cc08897aecb761eb0850c7e2/packages/api/src/typeDefs/collection.graphql#L2

Reply:
Sure! Nica catch 😄